common/app/layout/CollectionEmail.scala

```diff
@@ -0,0 +1,52 @@
+package layout
+
+import model.PressedPage
+import model.facia.PressedCollection
+import model.pressed.{CollectionConfig, PressedContent}
+
+import PartialFunction.condOpt
+
+object EmailContainer {
+
+  def fromPressedCollections(pressedCollections: List[PressedCollection]): List[EmailContainer] = {
```

Comment:
For me personally, I would have to squint at these functions for quite a while before realising that there was de-duplication across containers going on (though to be fair it's clear from reading the tests). Could we tweak the naming somehow or add a comment clarifying this?